.github/workflows/build-and-dockerize.yaml

```diff
@@ -79,6 +79,16 @@ jobs:
           file: ${{ github.workspace }}/${{ inputs.imageTag }}.zip
           destination: ${{ inputs.imageTag }}.zip
 
+      - name: upload app persisted documents artifact
+        uses: randomairborne/r2-release@v1.0.2
+        with:
+          endpoint: https://6d5bc18cd8d13babe7ed321adba3d8ae.r2.cloudflarestorage.com
+          accesskeyid: ${{ secrets.R2_ACCESS_KEY_ID }}
+          secretaccesskey: ${{ secrets.R2_SECRET_ACCESS_KEY }}
+          bucket: hive-js-build-artifacts
+          file: ${{ github.workspace }}/packages/web/app/src/gql/persisted-documents.json
+          destination: ${{ inputs.imageTag }}.app.documents.json
```

Comment:
We write this file here, so we can from within our deployment CI, fetch that file and publish it to the Hive registry/CDN.